test-unit.gemspec

```diff
@@ -36,9 +36,4 @@ Gem::Specification.new do |spec|
   }
 
   spec.add_runtime_dependency("power_assert")
-  spec.add_development_dependency("bundler")
-  spec.add_development_dependency("rake")
-  spec.add_development_dependency("yard")
-  spec.add_development_dependency("redcarpet")
-  spec.add_development_dependency("packnga")
```

Comment:
As far as I know, `add_development_dependency` is not preferable way now

* https://github.com/rubygems/bundler/pull/7222
* https://qiita.com/hirura/items/fe331672c8d41a1357e1 - ja

When specifying only in `Gemfile`, we can use `BUNDLE_WITHOUT` as https://github.com/test-unit/test-unit/pull/202/commits/325a61e3314ad5727b1b6fd6345f8b626a36ad7f#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R86-R87

Reply:
OK. I'm neutral for this.